tests/integration/assessments/test_pipeline_scheduler.py

```diff
@@ -0,0 +1,51 @@
+from pathlib import Path
+import pytest
+
+from datetime import datetime
+
+from databricks.labs.remorph.assessments.pipeline import PipelineClass
+from databricks.labs.remorph.assessments.scheduler.pipeline_scheduler import PipelineScheduler, \
+    get_hours_since_last_run, get_days_since_last_run
+from ..connections.helpers import get_db_manager
+
+
+@pytest.fixture()
+def extractor():
+    return get_db_manager("remorph", "postgres")
+
+
+@pytest.fixture(scope="module")
+def pipeline_config():
+    prefix = Path(__file__).parent
+    config_path = f"{prefix}/../../resources/assessments/scheduler/postgres_pipeline_config.yml"
+    config = PipelineClass.load_config_from_yaml(config_path)
+
+    for step in config.steps:
+        step.extract_source = f"{prefix}/../../{step.extract_source}"
+    return config
+
+
+def test_hours_since_last_run():
+    date_start_str = "2025-04-08 10:30:00"
+    date_format = "%Y-%m-%d %H:%M:%S"
+    datetime_start = datetime.strptime(date_start_str, date_format)
+    actual_hours = get_hours_since_last_run(datetime_start)
+    expected_hours = divmod((datetime.now() - datetime_start).total_seconds(),  3600)[0]
+    assert actual_hours == expected_hours, "The calculated hours since last run does not match the expected value."
+
+
+def test_days_since_last_run():
+    date_start_str = "2025-04-08 10:30:00"
+    date_format = "%Y-%m-%d %H:%M:%S"
+    datetime_start = datetime.strptime(date_start_str, date_format)
```

Comment:
Just use the constructor rather than parsing:
```suggestion
    datetime_start = dt.datetime(year=2025, month=4, day=8, hour=10, minute=30, second=0, tzinfo=dt.timezone.utc)
```